_posts/2021-01-01-Fortran-Newsletter-January-2021.md

```diff
@@ -0,0 +1,159 @@
+ [#278](https://github.com/fortran-lang/stdlib/pull/278) (WIP): Implementation of the `stdlib_stats_distribution` modules. It provides probability distribution and statistical functions.
+* [#284](https://github.com/fortran-lang/stdlib/pull/284) (WIP): Required changes to be able to use `stdlib` as a subproject in CMake
+* [#189](https://github.com/fortran-lang/stdlib/pull/189) (WIP): Initial implementation of sparse matrices.
+
+Don't hesitate to test and review these pull requests!
+
+Otherwise, ongoing discussions continue about usability of `stdlib`
+([#7](https://github.com/fortran-lang/stdlib/issues/7),
+[#215](https://github.com/fortran-lang/stdlib/issues/215),
+[#279](https://github.com/fortran-lang/stdlib/issues/279),
+[#280](https://github.com/fortran-lang/stdlib/issues/280),
+[#285](https://github.com/fortran-lang/stdlib/issues/285)),
+and new implementations for `stdlib`
+([#135](https://github.com/fortran-lang/stdlib/issues/135),
+[#212](https://github.com/fortran-lang/stdlib/issues/212),
+[#234](https://github.com/fortran-lang/stdlib/issues/234),
+[#241](https://github.com/fortran-lang/stdlib/issues/241),
+[#258](https://github.com/fortran-lang/stdlib/issues/258),
+[#259](https://github.com/fortran-lang/stdlib/issues/259),
+[#262](https://github.com/fortran-lang/stdlib/issues/262),
+[#268](https://github.com/fortran-lang/stdlib/issues/268),
+[#277](https://github.com/fortran-lang/stdlib/issues/277)).
+
+
+The candidate for file system operations to be included in stdlib is being developed by
+[@MarDiehl](https://github.com/MarDiehl) and [@arjenmarkus](https://github.com/arjenmarkus)
+in [this repository](https://github.com/MarDiehl/stdlib_os).
+Please try it out and let us know how it works, if there are any issues, or if the API can be improved.
+
+## Fortran Package Manager
+
+Here's what's new in `fpm`:
+
+
+TO DO
+
+Work in progress:
+
+TO DO
+
+`fpm` is still in early development and we need as much help as we can get.
+Here's how you can help today:
+
+* Use it and let us know what you think! Read the [fpm packaging guide](https://github.com/fortran-lang/fpm/blob/master/PACKAGING.md) to learn how to build your package with fpm, and the [manifest reference](https://github.com/fortran-lang/fpm/blob/master/manifest-reference.md) to learn what are all the things that you can specify in the fpm.toml file.
+* Browse the [open issues](https://github.com/fortran-lang/fpm/issues) and see if you can help implement any fixes or features.
+* Adapt your Fortran package for fpm and submit it to the [Registry](https://github.com/fortran-lang/fpm-registry).
+* Improve the documentation.
+
+The short-term goal of fpm is to make development and installation of Fortran packages with dependencies easier.
+Its long term goal is to build a rich and decentralized ecosystem of Fortran packages and create a healthy
+environment in which new open source Fortran projects are created and published with ease.
+
+## Compilers
+
+### Classic Flang
+
+TO DO
```

Comment:
```
There are several pull requests out for evaluation.

Only one pull request was merged in December:

* [PR#951 Fix for ICE in atomic instruction generation](https://github.com/flang-compiler/flang/pull/951)
```

Reply:
I'm not sure how to add a specific commit to this PR. So here are one brief Flang updates in the comments.

I left NVIDIA last Wednesday, and am no longer directly involved in the Flang projects. I am hoping that either Alexis Perry or someone at NVIDIA will take on the monthly Flang update duties here.

(I don't have permissions to push a commit Jeremie's branch and I'm too dumb to know if there is another way to do it.)